app/src/main/java/org/astraea/partitioner/nodeLoadMetric/ReceiverFactory.java

```diff
@@ -23,7 +23,7 @@ public ReceiverFactory() {
     beanCollector =
         BeanCollector.builder()
             .interval(Duration.ofSeconds(1))
-            .numberOfObjectsPerNode(10)
+            .numberOfObjectsPerNode(1)
```

Comment:
why?

Reply:
Because I optimized the algorithm, originally ten seconds of data were stored, and each calculation needs to be recalculated for each second of data. 
Now the data is stored each second, and the calculated results are recorded, and the results within ten seconds in total will be recorded.
The data calculated for ten seconds per second becomes the data calculated per second per second.